### PR TITLE
change service name to match readme

### PR DESCRIPTION
**Pull request by flosal-bccs**
Change is required for demo to work as per README instructions.

### CODE REVIEW

**Comment by germanattanasio** [comment]
I think the change should be in the README because the change to the manifest will break the deploy to bluemix

File: manifest.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ declared-services:
     plan: standard
 applications:
 - services:
-  - natural-language-classifier-standard
+  - natural-language-classifier-service
```
Comment (germanattanasio) [inline manifest.yml]: if we change this we need to change line 3.

**Comment by germanattanasio** [comment]
I'm happy to merge a fix for the README

**Comment by flosal-bccs** [comment]
Thank you for your feedback. I have reverted the initial change I made to the manifest file and edited the create service command, example training output, and sample manifest file to reflect the original manifest file.

**Comment by germanattanasio** [comment]
thanks @flosal-bccs !

**Pull request merged**